python/interactive_graph.py

```diff
@@ -0,0 +1,40 @@
+import matplotlib.pyplot as plt
+import pickle as pl
+import os
+
+# match with collision_checker.py
+COLLISION_RADIUS_KM = 0.1
+WARN_RADIUS_KM = 0.3
+
+
+def plot_min_distances(folder_name):
+
+    ABS_PATH_TO_THIS_FILE_DIRECTORY = os.path.dirname(os.path.abspath(__file__))
+    PATH_TO_PKL = os.path.join(ABS_PATH_TO_THIS_FILE_DIRECTORY, "../output/", folder_name, "minDistanceInteractive.pkl")
+
+    # Load figure from disk and display
+    fig_handle = pl.load(open(PATH_TO_PKL, 'rb'))
+    x, y = fig_handle.axes[0].lines[0].get_data()
+
+    # Getting warning times
+    i = 1  # i = 0 is y intercept, not wanted
+    warn_start_times = list()
+    while i < len(y):
+        if y[i] < WARN_RADIUS_KM:
+            warn_start_times.append(x[i])
+            while i + 1 < len(y) and y[i + 1] >= y[i]:
+                i += 1
+        i += 1
```

Comment:
Is there a better way of doing this? Seems pretty inefficient.

Reply:
Yeah good point. Probably not the biggest deal, but probably could have something like:

for i in range(1, len(y)):
    if y[i] < WARN and y[i-1] > WARN:
        warn_start_times.appned(i)